apps/prairielearn/src/pages/administratorFeatures/administratorFeatures.html.ts

```diff
@@ -303,7 +303,7 @@ export function AddFeatureGrantModalBody({
           ${(courses ?? []).map((course) => {
             return html`
               <option value="${course.id}" ${course.id === course_id ? 'selected' : ''}>
-                ${course.short_name}: ${course.title}
+                ${course.short_name}: ${course.title} (${course.repository ?? course.path})
```

Comment:
My thinking here: `repository` will generally be more useful, but it's nullable, so we show `path` as a backup.

Reply:
I think showing both seems like a lot of information here. I think we should be able to discern the correct course from having just either the repo or path. That being said, fine either way.

Reply:
I think that as we move to a BYOR model for repo git repos we are going to need to stop using the name of the repo as the path and instead use the course ID. Given that, I think we should just use the `course_id` here as well, especially given that this is an admin view and all we want is a way to clearly distinguish which course we are looking at?

An alternative way to distinguish courses would be to list the names of the owners. I think this might be especially helpful for institution admins, who will want to distinguish the legitimate CS 225 owned by Cinda from the fake one owned by some random student.

```suggestion
                ${course.short_name}: ${course.title} (${course.id})
```